bats/fb-test-backup.bats

```diff
@@ -58,6 +58,18 @@ BACKUP_ONLINE_DIR="${BACKUP_BASE_DIR}/online"
   tFileExists "${BACKUP_OFFLINE_DIR}/pulp_data.tar"
 }
 
+@test "check offline backup contents for Foreman Proxy Content" {
+  tForemanMaintainAvailable
+
+  if ! tPackageExists foreman && tPackageExists foreman-installer-katello; then
+    tFileExists "${BACKUP_OFFLINE_DIR}/pulp_data.tar"
+    tFileExists "${BACKUP_OFFLINE_DIR}/pgsql_data.tar.gz"
```

Comment:
We don't do pgsql_data.tar.gz for offline backups anymore ;)
See lines 43++ how to check that properly based on Foreman version (well, you probably need to check installer version, there).

Reply:
Makes sense, I had tested on Katello 4.13.